src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMNavigator.java

```diff
@@ -184,10 +183,10 @@ public void visitSources(@NonNull final SCMSourceObserver observer) throws IOExc
             List<Project> projects;
             if(gitlabOwner instanceof GitLabUser) {
                 // Even returns the group projects owned by the user
-                if(gitLabApi.getAuthToken().equals("")) {
-                    projects = gitLabApi.getProjectApi().getUserProjects(projectOwner, new ProjectFilter().withVisibility(Visibility.PUBLIC));
-                } else {
+                if(request.wantSubgroupProjects()) {
                     projects = gitLabApi.getProjectApi().getOwnedProjects();
```

Comment:
Not sure it makes sense to limit projects to what the Token has access to :thinking: Maybe better to use getUserProjects?

Reply:
We will discourage user to not add GitLab Access Token. This option is just a fallback. Generally, I expect user to add token.